config/initializers/100-logster.rb

```diff
@@ -10,12 +10,12 @@
 if Rails.env.development? && !Sidekiq.server? && ENV["RAILS_LOGS_STDOUT"] == "1"
   Rails.application.config.after_initialize do
     console = ActiveSupport::Logger.new(STDOUT)
-    original_logger = Rails.logger.chained.first
+    original_logger = Rails.logger.broadcasts.first
```

Comment:
`chained` is a `Logster::Logger` attribute, are we changing some behaviour here?

Reply:
Yes, because `Rails.logger` is not a `Logster::Logger` anymore with the changes introduced in Rails 7.1. The Logster logger is added to the broadcast logger, like any other logger.
You can see the change I made some time ago to Logster for Rails 7.1 compatibility: https://github.com/discourse/logster/commit/a023d778fb28bfa731865a49ca4383b1b50a9117